spec/models/event_registration_spec.rb

```diff
@@ -0,0 +1,45 @@
+require "spec_helper"
+
+describe EventRegistration do
+  it { should belong_to(:event) }
+  it { should belong_to(:user) }
+
+  describe "#github_stat_args" do
+
+    context "without github auth" do
+      it "returns nil" do
+        er = create(:event_registration)
+        expect(er.github_stat_args).to be_nil
+      end
+    end
+
+    context "with github auth" do
+      it "returns args" do
+        er = create(:er_for_october)
+
+        VCR.use_cassette("courte_user") do
+          expect(er.github_stat_args).to eq({user: "courte", day_begin: er.event.start_date, day_end: er.event.end_date})
```

Comment:
Line is too long. [121/80]<br>Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.